### ISSUE TITLE
Enable / disable production upgrades

### ISSUE THREAD

**Issue by func-vehicle** [func-vehicle/gtamanager#4]
Should be able to set owned upgrades of each business in case the user has not bought them all yet.

**Comment by ajkachnic** [comment]
Has any progress been made on this? I'm new to this project, but I'm happy to help

**Comment by func-vehicle** [comment]
This recent fork appears to be working on it https://github.com/juanminm/gtamanager

**Comment by starkythefox** [comment]
Hey there, I think I'm ready to send a pull request. Fixed a problem that was bugging me with the Bunker/MC business timers when you changed of upgrades due to stock/supplies being stored as time and not units like with Nightclub.

For the upgrades, there's the security upgrade added as well, although it's not used right now as there's no raid timer. Same goes with Nightclub's staff, there's no popularity.
For profiles from 1.9.0 and earlier I made it so that they have 2 upgrades for backward compatibility, but new profiles will be with no upgrades.
Also it contains a fix for the Bunker timer, which is 2h 20min or 700min for full stock with 2 upgrades according to most info sources, including the wiki.

The only think I'm not 100% sure if it will be correct is the one-upgrade-only MC supply timers. there's few information about them and I had to assume how much time is needed for a supply to be spent based on the other timers, the values seems to be the average none and two upgrades.
